disnake/widget.py

```diff
@@ -307,7 +309,7 @@ class Widget:
     name: :class:`str`
         The guild's name.
     channels: List[:class:`WidgetChannel`]
-        The accessible voice channels in the guild.
+        The accessible voice and stage channels in the guild.
```

Comment:
```suggestion
        The publicly accessible voice and stage channels in the guild.
```
or
```suggestion
        The voice and stage channels in the guild which are viewable by everyone.
```
or a combination of both.